hadoop-common-project/hadoop-auth/pom.xml

```diff
@@ -128,6 +128,10 @@
       <groupId>org.apache.zookeeper</groupId>
       <artifactId>zookeeper</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
```

Comment:
usually you should also add "snappy-java"

Reply:
Done. Thanks

Reply:
Hi,

Why did we decide to add `io.dropwizard.metrics` as part of this PR? Is it related with zk upgrade?

Reply:
Yea. They are needed only for the server side part in case you use that metrics provider, that's the default